### DISCUSSION TITLE
How to troubleshoot Star Wars: Squadrons (or any game, really)?

### DISCUSSION THREAD

**Discussion by JeffBerman** [Whisky-App/Whisky#881] [Q&A]
Hi there, I bought Star Wars: Squadrons mostly because someone here wrote that it worked well for them using Whisky. Unfortunately, it doesn't work for me on my M1 MacBook Pro. The settings screen comes up, then the intro video and music play fine (with a little flickering throughout). Then some words appear on the screen about Alderaan but after a few seconds the screen goes black and that's as far as it gets.

How do I go about troubleshooting this? Is there a post somewhere that explains how people usually try to get around these things?

Thanks!

**Comment by JeffBerman** [comment]
@LeaveNhA? Any thoughts?

**Reply by LeaveNhA** [reply]
Can I get a full system/sw info?
I'll try to help you bro.

**Reply by JeffBerman** [reply]
Sure!

It's a 2021 MacBook Pro: M1 Pro, 16 GB RAM, macOS 14.4.1.
Whisky 2.2.4, GPTK 1.4.0+0.

Here's the log:

```Whisky Version: 2.2.4
Date: 2024-04-02T21:08:30Z
macOS Version: 14.4.1

Bottle Name: Win10
Bottle URL: /Users/jeff/Library/Containers/com.isaacmarovitz.Whisky/Bottles/B2CA36C0-5115-4088-BFA1-6283B7326E96

Wine Version: 7.7.0
Windows Version: win10
Enhanced Sync: msync

Metal HUD: false
Metal Trace: false

Arguments: start /unix /Users/jeff/Library/Containers/com.isaacmarovitz.Whisky/Bottles/B2CA36C0-5115-4088-BFA1-6283B7326E96/drive_c/Program Files (x86)/Steam/steam.exe

Environment:
{
    "DXVK_ASYNC" = 1;
    WINEDEBUG = "fixme-all";
    WINEDLLOVERRIDES = "dxgi,d3d9,d3d10core,d3d11=b";
    WINEESYNC = 1;
    WINEMSYNC = 1;
    WINEPREFIX = "/Users/jeff/Library/Containers/com.isaacmarovitz.Whisky/Bottles/B2CA36C0-5115-4088-BFA1-6283B7326E96";
}

msync: bootstrapped mach port on wine-71dbd20-msync.
msync: up and running.
0100:err:setupapi:SetupDefaultQueueCallbackW copy error 1812 L"@C:\\windows\\system32\\drivers\\wineusb.sys,-1" -> L"C:\\windows\\inf\\wineusb.inf"
wine: configuration in L"/Users/jeff/Library/Containers/com.isaacmarovitz.Whisky/Bottles/B2CA36C0-5115-4088-BFA1-6283B7326E96" has been updated.
0130:err:ole:create_server class {294935ce-f637-4e7c-a41b-ab255460b862} not registered
037c:err:wininet:HTTP_ParseRfc1123Date unexpected time zone L"UTC"
0464:err:kerberos:kerberos_LsaApInitializePackage no Kerberos support, expect problems
0198:err:wininet:HTTP_ParseRfc1123Date unexpected time zone L"UTC"
0374:err:wininet:HTTP_ParseRfc850Date unexpected month format L" Apr 2024 07:53:00 UTC"
0374:err:wininet:HTTP_ParseDate unexpected date format L"Tue, 02 Apr 2024 07:53:00 UTC"
0374:err:wininet:HTTP_ParseDate unexpected date format L"Tue, 02 Apr 2024 07:53:00 UTC"
037c:err:wininet:HTTP_ParseRfc850Date unexpected month format L" Apr 2024 07:53:00 UTC"
```

Does that mean anything to you? Thanks for helping me!

**Reply by LeaveNhA** [reply]
I have a big problem in my home internet. Once I got it back, I'll start to download and try out for you.
Meanwhile, the log doesn't look like anything to me. (Maybe, I'm a wine-idiot)

**Reply by LeaveNhA** [reply]
In the second look. Can you try to play/change your date/time settings?

**Reply by JeffBerman** [reply]
Hi, sorry, I'm not sure what you're asking me to do. Change my computer's date and time to some other value and then try to play the game again?
